server/src/db/migrations/20220215182946_createHunterFlavorColumns.cjs

```diff
@@ -0,0 +1,25 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+const { knex } = require("../../models/Model");
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.table("hunters", (table) => {
+    table.string("description");
+    table.string("flavor");
+  });
+};
```

Comment:
I think having an id on these tables could serve you for later.

Reply:
This is just adding columns to a table that already exists. So there are IDs, you just don't see them in this schema. These are in the seeder so they will always be with the right data.